.travis.yml

```diff
@@ -13,6 +13,8 @@ matrix:
     env: TEST_TARGET=default
   - python: 3.5
     env: TEST_TARGET=coding_standards
+  - python: 3.6
+    env: TEST_TARGET=default
```

Comment:
It should work, no reason not to.

Reply:
Yeah, it passed. 

I figured I would merge this one, then we can make a `0.2.0` release